builtin/rebase.c

```diff
@@ -537,6 +537,7 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 		ACTION_SHOW_CURRENT_PATCH
```

Comment:
The commit message is too short, and the first line too long.

You could use something like "builtin rebase: show progress when run interactively"

Reply:
And by "use" I meant: for the oneline. The commit message still should explain what progress is meant (i.e. the one that shows how far along the todo list generation is).